### ISSUE TITLE
labels function returns non-existing labels for a node

### ISSUE THREAD

**Issue by xujiaxj** [RedisGraph/RedisGraph#2361]
Redisgraph version: 2.8.12 or see the following command:

```
>> MODULE LIST

1) 1) "name"
   2) "graph"
   3) "ver"
   4) (integer) 20812
```

RedisInsight version: 1.10.1, or see the bottom of the screenshot.

This screenshot says it all. Notice that the node with ID 183 has one label `Service` if I run the bottom query `match (n) where id(n)=183 return n`. If I run the `labels(n)` in the top query, it returns a list of `[Service, Namespace]`. I know for a fact that this node only has one label `Service` in my data.

<img width="1035" alt="image" src="https://user-images.githubusercontent.com/4541662/168692512-9400bf77-274a-459e-b365-86cfdae1c95c.png">

Not sure if this is related to https://github.com/RedisGraph/RedisGraph/issues/2357.

**Comment by swilly22** [comment]
@xujiaxj do you have steps for reproduction ?

**Comment by xujiaxj** [comment]
Yes this looks like it has been fixed in [2.8.13](https://github.com/RedisGraph/RedisGraph/releases/tag/v2.8.13)
